code/src/components/Album.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './album.css';
+import { Artist } from './Artist'
+import { Icon } from './Icon'
+
+// Körs en gång för varje varv i map, returnerar bild, artist coh album
+export const Album = props => {
+    return (
+        <section className="wrapper">
+            <article className="album">
+                <div className="image-container">
+                    <img src={props.album.images[1].url} className="cover-img" alt="" />
+                    <div className="icon-container">
+                        <Icon icon="./icons/heart.svg" />
+                        <a href={props.album.external_urls.spotify} className="play-icon">
+                            <Icon icon="./icons/play.svg" />
+                        </a>
+                        <Icon icon="./icons/dots.svg" />
+                    </div>
+                </div>
+
+                <a href={props.album.external_urls.spotify}>
+                    <h2>{props.album.name}</h2>
+                </a>
+                
+                <h3>
+                {props.album.artists.map(item => {
+                    return (
+                        <Artist 
+                            key={item.id} 
+                            name={item.name} 
+                            url={item.external_urls.spotify}
+                        />
+                    )
+                })}
+                </h3>
```

Comment:
This `h3` could be wrapping every single artist instead of all of them. Now you get the hover effect on all artists at the same time, _and_ it would improve the readability of this file if the `h3` was inside the `Artist` component instead

Reply:
I can't see it, when I hover over the artists it's only one artist at the time that gets darker. Or maybe I don't understand what you mean?

Reply:
When you hover one artist, all artists get underlined :)
<img width="217" alt="Skärmavbild 2020-11-24 kl  19 21 26" src="https://user-images.githubusercontent.com/51529201/100135665-413fd900-2e8a-11eb-9243-2270408c83a5.png">